src/main/resources/application-prod.yml

```
@@ -38,4 +38,10 @@ jwt:
   refresh-expiration: ${JWT_REFRESH_EXPIRATION:604800000}
 file:
   profile-image-dir: C:/Users/user/profile-images
-  profile-image-url-base: /uploads/profiles/
\ No newline at end of file
+  profile-image-url-base: /uploads/profiles/
+cloud:
+  aws:
+    region:
+      static: ${AWS_REGION:us-fake-1}
+    s3:
+      bucket: ${AWS_S3_BUCKET_NAME:fake-bucket-placeholder}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)